src/configure.rs

```diff
@@ -15,23 +15,19 @@ pub fn edit_configuration() {
 }
 
 fn get_editor() -> String {
```

Comment:
Totally nitpicky, but I would like to strip away the nested indentation here if possible.

I think something along these lines is more easily scannable:
```rust
fn get_editor() -> String {
    let editor = env::var("VISUAL").or_else(|_| env::var("EDITOR"));
    editor.unwrap_or_else(|_| STD_EDITOR.to_string())
}
```

Reply:
I'll get on it tomorrow (yay timezones)